oracle/price-feeder/oracle/oracle.go

```diff
@@ -277,10 +279,6 @@ func (o *Oracle) SetPrices(ctx context.Context) error {
 		return err
 	}
 
-	// TODO: make this more lenient to allow assigning prices even when unable to retrieve all
```

Comment:
doesnt the removal of this mean that if we have all but one whitelisted assets available, it would still send the remaining whitelisted assets without the one that isnt available?

Reply:
Now that we don't necessarily require the NON-whitelisted entries, it means that computedPrices may not have the same length, which is now legal.  The subsequent check ensures that all required are present.